tests/parametric/conftest.py

```diff
@@ -273,14 +273,14 @@ def java_library_factory(env: Dict[str, str], container_id: str, port: str):
 WORKDIR /client
 # COPY --from=apm_library_latest /dd-java-agent.jar ./tracer/
 # COPY --from=apm_library_latest /LIBRARY_VERSION ./tracer/
-RUN mkdir ./tracer/ && wget -O ./tracer/dd-java-agent.jar https://github.com/DataDog/dd-trace-java/releases/download/v1.12.1/dd-java-agent-1.12.1.jar
+RUN mkdir ./tracer/ && wget -O ./tracer/dd-java-agent.jar https://github.com/DataDog/dd-trace-java/releases/latest/download/dd-java-agent.jar
 COPY {java_reldir}/src src
 COPY {java_reldir}/build.sh .
+COPY {java_reldir}/run.sh .
 COPY {java_reldir}/pom.xml .
 COPY {protofile} src/main/proto/
 COPY binaries /binaries
 RUN bash build.sh
-COPY {java_reldir}/run.sh .
```

Comment:
Is there a particular reason for moving this line? Iterating on the run script will now do a build as well.

Reply:
Mainly because I iterate more over the code than the script 🤷 
But I understand why you moved it last then. And the script copy being faster than build the app, it feels okay to move it after then.